lib/feature_management.rb

```diff
@@ -16,4 +16,8 @@ def self.prefill_otp_codes?
   def self.proofing_requires_kbv?
     Figaro.env.proofing_kbv == 'true'
   end
+
+  def self.enable_dev_mode?
```

Comment:
I like the idea of giving this a more explicit name that relates to how we are using it. I must admit that the first time I read through this PR, before I read the PR description, I was super confused about what all of this was doing. how about `enable_translation_dev_mode` ?